packages/main/src/ListItem.ts

```diff
@@ -498,6 +510,10 @@ abstract class ListItem extends ListItemBase {
 		return !!this.deleteButton.length;
 	}
 
+	get hasIconSlot() {
```

Comment:
minor: 
"hasCustomIcon" is probably a more specific name...

But feel free to either reflect or discard this comment 

(I see that the convention hasXslot is already used in the file => I guess this is why you used that pattern, even though its name is not literally 100% accurate)